src/Mod/PartDesign/PartDesignTests/TestMirrored.py

```diff
@@ -69,24 +69,6 @@ def testMirroredPrimitiveCase(self):
         self.Doc.recompute()
         self.assertAlmostEqual(self.Mirrored.Shape.Volume, 2.0)
 
-    def testMirroredOffsetFailureCase(self):
```

Comment:
We don't have to drop this test function. Instead it can be renamed to testMirroredOffsetCase and "Invalid" must be changed to "Up-to-date".